### PR TITLE
Update issue templates, extend examples, minor improvements

### PR DESCRIPTION
**Pull request by stackdumper**
1. Update issue templates
2. Add more examples from https://prepack.io
3. Move `prepack` from `dependencies` to `peerDependencies` to allow the use of own `prepack` version
4. Set `webpack` peer dependency version to `^4` because package is only compatible with `webpack@^4`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "require": "babel-loader"
   },
   "dependencies": {
-    "prepack": "^0.2.34",
+    "react-native-iphone-x-helper": "^1.0.3",
```
Comment (gajus) [inline package.json]: Whats this for?

File: package.json [hunk 2]
```diff
@@ -60,7 +60,8 @@
     "sourceMap": false
   },
   "peerDependencies": {
-    "webpack": "^3 || ^4"
+    "prepack": "^0.2.48",
```
Comment (gajus) [inline package.json]: Whats the reason for making this a peerDependency?

Comment (stackdumper) [reply package.json]: To allow the providing of own version of `prepack`. For example, when a newer version is available.

File: package.json (hunk 1 above)
Comment (stackdumper) [reply package.json]: Added this accidentally

**Comment by stackdumper** [comment]
Any updates?

**Pull request merged**